### PR TITLE
Update .clang-format (make the instructions easier to read)

### PR DESCRIPTION
**Pull request by szb512**
I hope this is an improvement.

### CODE REVIEW

**Comment by szb512** [comment]
Log portion: 
autopkgtest [19:35:03]: test boot-smoke:  - - - - - - - - - - results - - - - - - - - - -
boot-smoke           PASS
autopkgtest [19:35:03]: @@@@@@@@@@@@@@@@@@@@ summary
timedated            PASS
hostnamed            PASS
localed-locale       PASS
localed-x11-keymap   PASS
logind               PASS
unit-config          PASS
storage              PASS
networkd-test.py     PASS
build-login          PASS
boot-and-services    PASS
udev                 PASS
root-unittests       PASS
upstream             FAIL timed out
boot-smoke           PASS

File: .clang-format [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # This configuration file can be used to auto-format the code base.
 # Not all guidelines specified in CODING_STYLE are followed, so the
 # result MUST NOT be committed indiscriminately, but each automated
-# change should be reviewed and only the appropriate ones commited.
+# change should be reviewed and only the appropriate ones can be commited.
```
Comment (keszybz) [inline .clang-format]: Can you drop this part please? The other two changes are OK.

Comment (szb512) [reply .clang-format]: Ok.

Comment (szb512) [reply .clang-format]: Fixed.

**Pull request merged**